encryption_works.md

```diff
@@ -98,25 +98,33 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
 
 ## Anonymize Your Location with Tor
 
-Tor is a software service that allows you to use the Internet while concealing your IP address, which is, in general, a fairly accurate representation of your location. The [Tor](https://www.torproject.org/) network is made up of over 3,600 volunteer servers called nodes. When someone uses the Tor network to visit a website their connection gets bounced through three of these nodes (called a circuit) before finally exiting into the normal Internet. Anyone intercepting traffic will think your location is the final node which your traffic exits from.
+Despite NSA surveillance, it’s still possible to browse the web with a high degree of anonymity. The Tor Browser is a web browser like Firefox or Internet Explorer, but it allows you to use the Internet while concealing your IP address, which is generally an accurate representation of your approximate location. 
```

Comment:
you should mention that it's based off of the Firefox browser, even.  I would sub IE for Safari because IE is yucky (and Safari suffices for not-alienating-people...) but that's just my stylistic choice :)